src/pages/Categories.js

```diff
@@ -1,12 +1,10 @@
 import React from 'react';
 import List from '../components/List';
 
-const Categories = () => {
-  return (
-    <div>
-      <List />
-    </div>
-  );
-};
+const Categories = () => (
+  <div>
+    <List />
+  </div>
```

Comment:
- [ ] **Requirement**  Categories

* should display only a button with text "Check status"